composer.json

```diff
@@ -9,6 +9,7 @@
     "spryker-eco/avalara-tax-extension": "1.0.0",
     "spryker/calculation-extension": "^1.0.0",
     "spryker/cart-extension": "^4.0.0",
+    "spryker/checkout-rest-api-extension": "^1.0.0",
```

Comment:
Please add `"spryker/transfer": "^3.25.0",` in case using `->getOrFail` methods.

Reply:
I've already added it in another PR https://github.com/spryker-eco/avalara-tax/pull/4